src/modules/updateCheckboxStatus.js

```diff
@@ -0,0 +1,11 @@
+import tasks from './array';
+
+const updateCheckboxStatus = (status, index) => {
+  if (status === true) {
+    tasks[index - 1].completed = true;
+  } else if (status === false) {
+    tasks[index - 1].completed = false;
+  }
+};
```

Comment:
- [ ] It will be great to save the updated objects into the local storage. What i mean by this is that when a user clicks on the check box the object stored in the local storage should update also to true or false. This will ensure that even when the page reloads, the user has the correct data

Reply:
Hi @peterrobert , since on refreshing the page, the checkbox would get unchecked (completed: false), which is a default value, I haven't called the local storage function here. But as soon as the user clicks on the clear all completed button, setLocalStorage function gets called and data is updated.

If you mean, on refreshing the page, if the checkbox is checked, it should remain the same. This functionality is neither in the original(archived)  website nor in the video demonstration.

Reply:
What's the purpose of doing that? Suppose, I check one task, and update it in local storage. Now I refresh the page, the task is completed because it fetches from the Ls. Now even without checking the task, I click on the ClearAllcheckeditems, and it is going to clear that particular task.